src/middlewared/middlewared/plugins/mdns.py

```diff
@@ -738,12 +738,8 @@ def __init__(self, **kwargs):
         super(mDNSServiceHTTPThread, self).__init__(**kwargs)
 
     def setup(self):
-        webui = self.middleware.call_sync('datastore.query', 'system.settings')
-        if not webui[0]['stg_guihttpsredirect']:
-            self.port = int(webui[0]['stg_guiport'] or 80)
-        else:
-            self.port = 6000
-        self.regtype = "_http._tcp."
+        self.port = 6000
```

Comment:
`_http._tcp.` is WebUI, it should be port 80 (and advertised only if redirect is disabled)